### PR TITLE
Implement save movies

### PR DESCRIPTION
**Pull request by sddenuwara**

### CODE REVIEW

File: tests/e2e/test_create_movies_page.py [hunk 1]
```diff
@@ -1 +1,104 @@
 # TODO: Feature 2
+from app import app
+from src.repositories.movie_repository import get_movie_repository
+movie_repository = get_movie_repository()
+
+# Case 1: app.get('/movies/new') should return a 200 status code
```
Comment (Jacopo-Magnani) [inline tests/e2e/test_create_movies_page.py:6]: Good job with testing worst case scenarios

File: tests/unit/test_create_movie.py [hunk 2]
```diff
@@ -1 +1,106 @@
...
+
+    test_movie = movie_repository.create_movie('Hunger Games', 'Gary Ross', 'abc')
+    assert test_movie.title == 'Hunger Games'
+    assert test_movie.director == 'Gary Ross'
+    assert test_movie.rating == 'abc'
+    assert len(movie_repository.get_all_movies()) == 1
+
+    movie_repository.clear_db()
+
+# Case 4c: Rating out of range (>5) should create a movie
```
Comment (Jacopo-Magnani) [inline tests/unit/test_create_movie.py:80]: Could you combine  Case 4c and 4d

File: app.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-from flask import Flask, redirect, render_template
+from flask import Flask, request, redirect, render_template
```
Comment (Jacopo-Magnani) [inline app.py:1]: Change to: from flask import Flask, redirect, render_template, request

File: tests/unit/test_create_movie.py [hunk 4]
```diff
@@ -1 +1,106 @@
...
+
+    test_movie = movie_repository.create_movie('Hunger Games', None, 3)
+    assert test_movie.title == 'Hunger Games'
+    assert test_movie.director == None
+    assert test_movie.rating == 3
+    assert len(movie_repository.get_all_movies()) == 1
+
+    movie_repository.clear_db()
+
+# Case 4a: No rating should create a movie
```
[the remark this answers is not in the document]
Comment (sddenuwara) [reply tests/unit/test_create_movie.py:52]: I would like to leave them separate to ensure that each case can be identified from each other during testing

**Pull request merged**